### DISCUSSION TITLE
Firewall Rules - Is toggling via command line possible?

### DISCUSSION THREAD

**Discussion by TFWol** [sandboxie-plus/Sandboxie#2901] [Q&A Feedback] [answered]
There are many occasions where I would like to quickly disable/enable internet to a program/group in a sandbox right before starting it.

Is there a built-in method to toggle a defined firewall rule via commandline like below?
```
start.exe /box:<boxname> /fw toggle "<rule name>"
```
Currently I have to rely on a script I made to parse the .ini to manipulate the entry. It was pretty painful to make for just one of the entries.

**Comment by bastik-1001** [comment]
Sounds like a feature request, if that can't be done, already, which I can't tell to be the case.

**Reply by isaak654** [reply, marked as answer]
Solution provided by @TFWol: https://github.com/sandboxie-plus/Sandboxie/issues/2939#issuecomment-1565037973
